src/shogun/features/Alphabet.cpp

```diff
@@ -832,6 +832,14 @@ void CAlphabet::translate_from_single_order_reversed(ST* obs, int32_t sequence_l
 	}
 }
 
+CSGObject * CAlphabet::clone()
+{
+	CAlphabet * alph_clone = (CAlphabet *) CSGObject::clone();
+	alph_clone->init_map_table();
+	alph_clone->copy_histogram(this);
```

Comment:
I was hoping you guys could tell! There was that obscure comment in the code [here](https://github.com/shogun-toolbox/shogun/blob/develop/src/shogun/features/Alphabet.cpp#L740), I can't say for sure the patch is enough. Certainly the code before wasn't all that was needed :P